src/cli.cc

```diff
@@ -59,19 +76,33 @@ ExitCode Install(AkliteClient& client, int version) {
 }
 
 ExitCode CompleteInstall(AkliteClient& client) {
+  ExitCode exit_code{ExitCode::UnknownError};
   TufTarget target{client.GetPendingTarget()};
 
   if (target.IsUnknown()) {
     LOG_INFO << "There is no pending installation to complete";
     return ExitCode::NoPendingInstallation;
   }
 
-  if (client.CompleteInstallation()) {
-    return ExitCode::Ok;
+  auto install_res = client.CompleteInstallation();
+  switch (install_res.status) {
+    case InstallResult::Status::Ok: {
+      exit_code = ExitCode::Ok;
+      break;
+    }
+    case InstallResult::Status::NeedsCompletion: {
+      // Update of sotree and Apps have been completed successfully;
+      // bootloader was updated too and it requires device reboot to confirm its update.
+      exit_code = ExitCode::InstallNeedsRebootForBootFw;
+      break;
+    }
+    default:
+      // TODO
+      break;
```

Comment:
I don't get why is there a `// TODO` comment in all the default switch clauses.
Shouldn't we core dump or return `UnknownError` in these cases?